helm/scaphandre/templates/daemonset.yaml

```diff
@@ -53,8 +53,9 @@ spec:
           name: powercap
           readOnly: false
       securityContext:
```

Comment:
@CalmaIndustry I have the following error:

```
scaphandre on  fix/add-privileged [!] is 📦 v1.0.0 via 🦀 v1.81.0 
❯ helm upgrade --install scaphandre helm/scaphandre
W0925 11:24:46.489160   84959 warnings.go:70] unknown field "spec.template.spec.securityContext.privileged"
Release "scaphandre" has been upgraded. Happy Helming!
NAME: scaphandre
LAST DEPLOYED: Wed Sep 25 11:24:45 2024
NAMESPACE: default
STATUS: deployed
REVISION: 4
TEST SUITE: None
```

IMHO there is an indentation issue, and `securityContext` should be in the container item block, not in the `containers` array:

```yml
    spec:
      containers:
      - name:  {{ template "scaphandre.name" . }}
        image: "{{ .Values.image.name }}:{{ .Values.image.tag }}"
        args:
            {{- range $key, $value := .Values.scaphandre.args }}
            {{- if $value }}
            - --{{ $key }}={{ $value }}
            {{- else }}
            - --{{ $key }}
            {{- end }}
            {{- end }}
            - {{ .Values.scaphandre.command }}
            {{- range $key, $value := .Values.scaphandre.extraArgs }}
            {{- if $value }}
            - --{{ $key }}={{ $value }}
            {{- else }}
            - --{{ $key }}
            {{- end }}
            {{- end }}
        env:
        {{- if .Values.scaphandre.rustBacktrace }}
        - name: RUST_BACKTRACE
          value: '{{ .Values.scaphandre.rustBacktrace }}'
        {{- end }}
        ports:
        - name: metrics
          containerPort: {{ .Values.port }}
        resources:
{{ toYaml .Values.resources | indent 10 }}
        volumeMounts:
        - mountPath: /proc
          name: proc
          readOnly: false
        - mountPath: /sys/class/powercap
          name: powercap
          readOnly: false
        securityContext:
          privileged: {{ .Values.securityContext.privileged }}
          runAsUser: {{ .Values.securityContext.userID }}
          runAsGroup: {{ .Values.securityContext.userGroup }}
```

If I change this, it installs fine:

```
scaphandre on  fix/add-privileged [!] is 📦 v1.0.0 via 🦀 v1.81.0 
❯ helm upgrade --install scaphandre helm/scaphandre
Release "scaphandre" has been upgraded. Happy Helming!
NAME: scaphandre
LAST DEPLOYED: Wed Sep 25 11:30:01 2024
NAMESPACE: default
STATUS: deployed
REVISION: 6
TEST SUITE: None
```

Then, the `scaph_host_power_microwatts` metrics started to report non 0/NaN values for at least one instance:

![image](https://github.com/user-attachments/assets/38fdd2a1-9383-4ec0-a98d-c54d66648408)

I am going to check why the other instances report 0.

Reply:
> I am going to check why the other instances report 0.

After deleting the pods, all the new pods run as expected without errors in the logs. And the metrics are available.

![Screenshot from 2024-09-25 13-08-44](https://github.com/user-attachments/assets/4edc82c7-0d3c-41db-8a70-e911f4385076)

As far as I can tell, there is not even the need to do a `chown` or `chmod` on the host machine.

So the only remaining problem is [this](https://github.com/hubblo-org/scaphandre/pull/391#discussion_r1774906974).